src/main/kotlin/no/nav/syfo/application/OppfolgingsoppgaveService.kt

```diff
@@ -46,18 +48,33 @@ class OppfolgingsoppgaveService(
     fun editOppfolgingsoppgave(
```

Comment:
Synes denne måten å gjøre det på virker logisk! Kunne også vurdert å ha en liten kommentar over funksjonen om det er noe vi ønsker å dokumentere tydelig. Er jo litt funky regel.
```suggestion
/**
 * God forklaring på hva som skjer og hvorfor om vi får inn endret oppfølgingsgrunn
 */
    fun editOppfolgingsoppgave(
```

Reply:
Et alternativ til kommentar (kan ha begge og selvfølgelig) ville vært å trekke følgende ut i egen metode:
```kotlin
if (newOppfolgingsgrunn != null && existingOppfolgingsoppgave.sisteVersjon().oppfolgingsgrunn != newOppfolgingsgrunn) {
```

Reply:
Har refaktorert til egen metode nå. Si fra om dere har bedre forslag til navn og plassering :) Kan også forenkle denne noe når vi kan gjøre newOppfolgingsgrunn påkrevd etter frontend er tilpasset.

Reply:
Hvis jeg skulle valgt så ville jeg lagt den i `Oppfolgingsoppgave`, men jeg er litt usikker på om det bryter med måten det blir gjort ellers.

Så da ville det blitt følgende i oppfølgingsoppgave:
```kotlin
fun isOppfolgingsgrunnEqualTo(newOppfolgingsgrunn) = sisteVersjon().oppfolgingsgrunn == newOppfolgingsgrunn
```

Om det ikke gir mening å flytte den så tror jeg kanskje jeg hadde endret ordlyden til metoden du lagde til noe tilsvarende som forslaget ovenfor.

Reply:
Ja, kan se på ytterligere refaktorering i neste PR som gjør `newOppfolgingsgrunn` påkrevd.